doctr/datasets/datasets/base.py

```diff
@@ -100,7 +100,6 @@ def __init__(
             archive_path = Path(archive_path)
             dataset_path = archive_path.parent.joinpath(archive_path.stem)
             if not dataset_path.is_dir() or overwrite:
-                with ZipFile(archive_path, 'r') as f:
-                    f.extractall(path=dataset_path)
+                shutil.unpack_archive(archive_path, dataset_path)
```

Comment:
nice switch

Reply:
@fg-mindee 
yes and it can unpack all formats tar.gz, tar.bz, zip, ..... but i have with the folder check a little problem as you can see in the failed tests

Reply:
solved